website/content/docs/import/azurekv.mdx

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+page_title: Azure secret import source
+description: The Azure KeyVault source imports secrets from Azure to Vault.
+---
+
+# Azure secret import source
+
+Use the Azure source to import secret data from Azure KeyVault into your Vault instance.
+
+## Argument reference
+
+Refer to the [HCL syntax](/vault/docs/import#hcl-syntax-1) for arguments common to all source types.
+
+## Additional arguments
+
+Required:
+
+- `key_vault_uri` `(string)` - The URI of the Azure KeyVault to import from
+
+All the following are optional, in which case the default provider mechanisms are used, e.g.
+a preliminary `az login` can suffice.
```

Comment:
```suggestion
The following parameters are optional. If you leave these parameters unset, for example, by using `az login` before importing, Vault uses the default provider mechanisms.
```
I wasn't entirely sure what we're trying to say here. Let me know if this isn't an appropriate edit and I'll rework it.